internal/fnruntime/fnerrors.go

```diff
@@ -64,3 +59,15 @@ func (fe *ExecError) String() string {
 func (fe *ExecError) Error() string {
 	return fe.String()
 }
+
+// StderrNotEmpty is an error type which indicates that the function
```

Comment:
Conceptually, this is NOT an error type.

Reply:
Removed this type